module/data/item/fields/enchantment-field.mjs

```diff
@@ -167,16 +177,6 @@ export class EnchantmentData extends foundry.abstract.DataModel {
     return errors.length ? errors : true;
   }
 
-  /* -------------------------------------------- */
-
-  /**
-   * Fetch the tracked enchanted items.
-   * @returns {Promise<Item5e[]>}
-   */
-  async getItems() {
```

Comment:
Just on the item sheet for originally displaying this list, but since I was able to rework this to not be async I could get rid of the old method and rely just on the new property.